R/board_url.R

```diff
@@ -141,12 +179,153 @@ pin_store.pins_board_url <- function(board, name, paths, metadata,
   abort_board_read_only("board_url")
 }
 
+#' @export
+pin_version_delete.pins_board_url <- function(board, name, version, ...) {
+  abort_board_read_only("board_url")
+}
+
 #' @export
 write_board_manifest_yaml.pins_board_url <- function(board, manifest, ...) {
   abort_board_read_only("board_url")
 }
 
 # Helpers ------------------------------------------------------------------
+
+board_url_error_message <- c(
+  "{.var urls} must resolve to either:",
```

Comment:
I didn't see this error tested in the snapshots. I think it's going to make a rather long error.